.lando.yml

```diff
@@ -2,22 +2,25 @@ name: conifer
 recipe: wordpress
 config:
   webroot: wp
-  php: '8.2'
+  php: '8.1'
 
 services:
-  node:
-    type: node:14
 
   appserver:
-    run_as_root:
+    build_as_root:
       - apt-get update
       - apt-get install zip
       - apt-get install subversion -y
 
-    run:
+    build:
```

Comment:
Can I get some clarification from you on why this was changed from `run` to `build`? I'm running into issues setting up conifer locally, and I'm pretty sure this change is the reason. According to the [Lando docs on build steps](https://docs.lando.dev/services/lando-3.html#build-steps), the `run` step runs after the service has booted up while the `build` step runs before the service has booted up. I think we need to wait to set up WP until after the service has booted up. Is this working on your end?